pkg/descheduler/descheduler.go

```diff
@@ -97,7 +97,7 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 	wait.Until(func() {
 		nodes, err := nodeutil.ReadyNodes(ctx, rs.Client, nodeInformer, nodeSelector, stopChannel)
 		if err != nil {
-			klog.V(1).Infof("Unable to get ready nodes: %v", err)
+			klog.InfoS("Unable to get ready nodes", "err", err)
```

Comment:
```suggestion
			klog.V(1).InfoS("Unable to get ready nodes", "err", err)
```

Reply:
@farah can you please address this review comment? Thanks!

Reply:
sorry for the oversight. done.